test_unstructured_ingest/test-ingest-s3.sh

```diff
@@ -5,11 +5,15 @@ set -e
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 cd "$SCRIPT_DIR"/.. || exit 1
 
-if [[ "$(find test_unstructured_ingest/expected-structured-output/s3-small-batch/ -type f -size +20k | wc -l)" -ne 3 ]]; then
-    echo "The test fixtures in test_unstructured_ingest/expected-structured-output/ look suspicious. At least one of the files is too small."
-    echo "Did you overwrite test fixtures with bad outputs?"
-    exit 1
-fi
+# NOTE(robinson) - uncomment the section below and regenerate test fixtures once
+# the ONNXRuntimeError issue in unstructured-inference is resolved
+# https://github.com/Unstructured-IO/unstructured-inference/issues/116
+
+# if [[ "$(find test_unstructured_ingest/expected-structured-output/s3-small-batch/ -type f -size +20k | wc -l)" -ne 3 ]]; then
```

Comment:
We can revert this change now. This was a temporary work around from when we hit the blank page issue with ONNX

Reply:
Reverted in [9db31f2](https://github.com/Unstructured-IO/unstructured/pull/662/commits/9db31f2dca3ac5b0982f8c5e6c4835c9888d499f).